manuscript/applications.tex

```diff
@@ -1,36 +1,50 @@
-\section{Other Uses of Dynamic Deployment}
 The concept of dynamic demand-driven deployment is not new,
 and has been used in various applications. Various algorithms
 are used to optimize deployment of facility or utilization
 of given sources.
 
-\subsection{Aircraft Dispatch}
 To maximize fleet utilization and minimize
 operating costs, airlines predict future demands
-and optimize their flight schedule and aircraft
-type using linear programming methods \cite{berge_demand_1993}. The Demand
-Driven Dispatch process evaluates benefits and costs
-in various stages of an airline planning stage, like
-pricing, aircraft assignment, and crew scheduling \cite{shebalov_practical_2009}.
+and optimize their flight schedules and aircraft
+types using linear programming methods \cite{berge_demand_1993}. The Demand
+Benefits and costs of ticket pricing, aircraft assignment, and crew 
+scheduling are evaluated with a linear optimization method called
+ ``Demand Driven Dispatch'' \cite{shebalov_practical_2009}, a type
+of deterministic optimization method.
 
-\subsection{Timber Industry}
 A sawmill facility that processes timber contains 
 many steps, from sawing, drying to planning. An
 agent-based simulation is used to analyse demand-
-driven production planning \cite{yáñez_agent-based_2009}.
+driven production planning for a manufacture facility
+in Eastern Canada \cite{yanez_agent-based_2009}.
 The facility unit can be represented by a group of many facilities,
 for example, source, sawing operations, drying operations, warehouse,
-make, and delivery. In the simulation, each agents are given their parameters
+make, and delivery. In the simulation, each agent is given its parameters
 and model its procedures.
 
-The demand agent would communicate to the make agent of the demand plan,
+The demand agent communicates to the make agent of the demand plan,
```

Comment:
"communicates the demand plan, <explain what a demand plan is>, to the make agent."